UI/js-src/lsmb/menus/Cache.js

```diff
@@ -0,0 +1,65 @@
+define(["dojo/_base/lang","dojo/when" /*=====, "dojo/_base/declare", "dojo/store/api/Store" =====*/],
+function(lang, when /*=====, declare, Store =====*/){
+
+// module:
+//		lsmb/menus/Cache
+
+var Cache = function(masterStore, cachingStore, options){

```

Comment:
Why do we need our own Cache store? Isn't the Dojo cache store good enough? If not, please include documentation why not.